public/main.js

```diff
@@ -50,13 +51,20 @@ function makeButton(content, cssClass, onClick) {
     return button;
 }
 
+function countIncompletes(todos, countLabel) {
+    countLabel.textContent = todos.filter(function(todo) {
```

Comment:
Nice use of higher order functions here! I think in this case, `reduce` would be more appropriate.